src/metrics/ms_run_metrics.nf

```diff
@@ -11,9 +11,13 @@ python_image = 'mpc/nextqcflow-python:latest'
 // Set if you want to extract specific headers from Thermo raw measurements, otherwise the default is used.
 // Have a look into the corresponding python script for the headers.
 params.ms_run_metrics__thermo_headers = ""
```

Comment:
The params should be defined in the main.nf and passed down.
Anyways: the params for the extraction should go into the central params file (as tehy are machine specific)

The params for the memory etc. into the main.nf